stylix/hm/icon.nix

```diff
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+let
+	cfg = config.stylix.iconTheme;
+	polarity = config.stylix.polarity;
+in {
+	imports = [ ../icon.nix ];
+	config = lib.mkIf (config.stylix.enable && cfg.enable && pkgs.stdenv.hostPlatform.isLinux) {
+		gtk = {
+			# inherit (cfg) enable;
```

Comment:
Is this a debugging leftover or is this meant to be removed?

Reply:
Ya this was a leftover from me testing some stuff before, removing it in the next commit.